cypress/support/api_commands.js

```diff
@@ -0,0 +1,7 @@
+
+Cypress.Commands.add('api_retrieveCustomers', (page = 1, limit = 10, size = 'All') => {
+    cy.request({
+        method: 'GET',
+        url: `http://localhost:3001/customers?$page=${page}&limit=${limit}&size=${size}`
```

Comment:
Here, instead of hardcoding the API URL, it could come from `Cypress.env('API_URL')`.

This would allow the same test to be run against different API environments, which could have different URLs.

Reply:
Thanks Walmyr, for some reason when I tried using my` _backendUrl_` it didn't work. I have since updated my `config.js` file with your suggestion, and updated this `api_commands.js` and now it all works!